file.go

```diff
@@ -77,6 +77,16 @@ func (e *FileError) Error() string {
 	return fmt.Sprintf("%s %s", e.FieldName, e.Msg)
 }
 
+// Format standard of X9.37 specification used to parse file
+type Format uint32
+
+const (
+	// Discover format
+	Discover Format = iota
```

Comment:
What if we renamed this to `Detect`? I'm worried Discover (the company) might have an X9 format we run into one day...

Reply:
@adamdecaf Thinking of dropping the Format enum. It's too restrictive.Unfortunately FRB adoption of DTSU requires EBCDIC encoding. I'm making some changes to allow reading/writing with configurable options. It should allow more flexibility for others.

Reply:
Okay. We'll write up a matrix to explain the known combinations.